docs/pages/database-access/guides/redshift-serverless.mdx

```diff
@@ -0,0 +1,402 @@
+</Admonition>
+
+### Bootstrap access to Redshift Serverless
+
+Teleport can bootstrap IAM permissions for the Database Service based on its
+configuration using the `teleport db configure bootstrap` command. You can use
+this command in automatic or manual mode:
+
+- In automatic mode, Teleport will attempt to create appropriate IAM policies
+  and attach them to the specified IAM identity (user or role). This requires the
+  IAM permissions we applied to the `teleport-redshift-serverless-node` role earlier.
+- In manual mode, Teleport will print required IAM policies. You can then create
+  and attach them manually using the AWS management console.
+
+<Admonition type="note" title="AWS credentials">
+AWS Credentials are only required if you’re running the command in "automatic"
+mode. The command uses the default credential provider chain to find AWS
+credentials. See [Specifying Credentials](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials)
+for more information.
+</Admonition>
+
+Run one of the following commands on your Database Service node:
+
+<Tabs>
+<TabItem label="Automatic">
+
+Use this command to bootstrap the permissions automatically when your Teleport
+Database Service runs on a host with AWS credentials. Teleport will automatically
+determine the available role/user, or you can specify one with `--attach-to-role`
+or `attach-to-user`.
+
+```code
+$ teleport db configure bootstrap -c /etc/teleport.yaml
+```
+
+</TabItem>
+<TabItem label="Manual">
+
+Use this command to display required IAM policies which you will then create in your AWS console:
+
+```code
+$ teleport db configure bootstrap -c /etc/teleport.yaml --manual
+```
+
+</TabItem>
+</Tabs>
+
+See the full `bootstrap` command
+[reference](../../database-access/reference/cli.mdx#teleport-db-configure-bootstrap)
+for more information.
+
+### Start the Database service
+
+Run the following command on the Database Service node, depending on how you
+installed Teleport:
+
+<Tabs>
+<TabItem label="Package Manager">
+
+```code
+$ sudo systemctl start teleport
+```
+
+</TabItem>
+<TabItem label="TAR archive">
+
+```code
+$ sudo teleport install systemd --output=/etc/systemd/system/teleport.service;
+$ sudo systemctl enable teleport
+$ sudo systemctl start teleport
+```
+
+</TabItem>
+</Tabs>
+
+The Database Service will discover all Redshift databases according to the
+configuration and register them in the cluster. The Database Service will also
+attempt to configure IAM access policies for the discovered databases. Keep in
+mind that AWS IAM changes may not propagate immediately and can take a few minutes
+to come into effect.
+
+<Admonition type="note" title="AWS credentials">
+
+The Teleport Database Service uses the default credential provider chain to
+find AWS credentials. See
+[Specifying Credentials](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials)
```

Comment:
Is this a repeat of the "Specifying credentials" step from earlier?

Would it make more sense to have a step where you confirm that you have supplied the correct AWS credentials on the Database Service host? Also, since the host is an EC2 instance, should we tell the reader to use instance metadata for credentials?

Reply:
I think the former. This was brought in from another guide I used as a template, or written before the other admonition, I can't remember. In any case, I'm just gonna `rm` it.